packages/fetch-engine/src/__tests__/node-api.test.ts

```diff
@@ -10,16 +10,17 @@ const CURRENT_BINARIES_HASH = enginesVersion
 
 jest.setTimeout(120_000)
 
-describe('download', () => {
+const describeIf = (condition: boolean) => (condition ? describe : describe.skip)
+const isNodeAPI = process.env.PRISMA_CLI_QUERY_ENGINE_TYPE !== 'binary'
+
+describeIf(isNodeAPI)('download', () => {
```

Comment:
Is this really NodeAPI specific or does this just guarantee that this is only ran once?